books/free-programming-books-hu.md

```diff
@@ -35,7 +35,7 @@
 
 ### Arduino
 
-* [Arduino programozási kézikönyv](http://avr.tavir.hu) - Brian W. Evans írása alapján fordította, kiegészítette és frissítette Cseh Róbert (PDF - regisztráció szükséges)
```

Comment:
not sure what this means

Reply:
... `based on her writing, it was translated, supplemented and updated by Róber Cseh`

Reply:
Recovered at 0a4dbc179e4b2f6346fc15579d7c5edec1b3433e